src/command.cc

```diff
@@ -198,6 +199,56 @@ AttrPathMixin::fixupAttrPath()
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+  argparse::Argument &
+RegistryFileMixin::addRegistryFileArg( argparse::ArgumentParser & parser )
+{
+  return parser.add_argument( "--registry-file" )
+               .help(
+                  "The path to the 'registry.json' file."
+               )
+               .required()
+               .metavar( "PATH" )
+               .action( [&]( const std::string & strPath )
+                        {
+                          this->setRegistryPath( strPath );
+                        }
+                      );
+}
+
+  void
+RegistryFileMixin::setRegistryPath(const std::filesystem::path & path)
+{
+  this->registryPath = path;
```

Comment:
nit: doesn't seem like we really need a setter

Reply:
Addressed above